plugins/generator-1.19.4/forge-1.19.4/procedures/entity_get_percent_frozen.java.ftl

```diff
@@ -0,0 +1 @@
+(${input$entity}.getPercentFrozen())
```

Comment:
This should have a float marker (`/*@float*/`) at the very beginning (same in 1.20.1)

Reply:
Done